### ISSUE TITLE
Validation loss always 0.

### ISSUE THREAD

**Issue by ArpithaFAU** [keras-team/keras#14296]
I am training my network with mixed input data (numeric and images). However, during traing the validation loss remains always 0. I tried different loss functions but have the same problem.

I tried to use the solution from https://github.com/tensorflow/tensorflow/issues/42053 , but it does not work and also i am using stand alone keras not tf.keras.

Can someone provide with some suggestions?
tf-version = 2.1.0
keras version = 2.3.1

**Comment by jvishnuvardhan** [comment]
@ArpithaFAU It is strange.  Are you using funtional model with two input branches? Can you share a standalone code to reproduce the issue? Thanks!

**Comment by jvishnuvardhan** [comment]
This is a stale issue. I am assuming this was got resolved already. If not, please reopen with a standalone code. Thanks!

**Comment by CJSeqtek** [comment]
I get the similar error, any update on this?

**Comment by njoppi2** [comment]
i have the same problem

**Comment by njoppi2** [comment]
Try changing the loss in your model.fit from `loss="categorical_crossentropy"` to `loss="binary_crossentropy"`

I Had the SAME problem, but this worked for me. The thing is that I have a binary classification model, with only 1 output node, not a multi-classification model with multiple output node, so `loss="binary_crossentropy"` is the appropriate loss function in this case.

😊
